src/view/uniprotkb/components/SequenceView.tsx

```diff
@@ -102,6 +104,31 @@ export const IsoformInfo: React.FC<{ isoformData: Isoform }> = ({
         .map(syn => syn.value)
         .join(', '),
     },
+    {
+      title: 'Differences from canonical',
+      content: isoformData.varSeqs && !!isoformData.varSeqs.length && (
+        <ul className="no-bullet">
+          {isoformData.varSeqs.map(varSeq => (
```

Comment:
Minor stylistic comment: could destructure `varSeq` here to slightly shorten the object paths in the body of the map.